content/webapp/pages/page.js

```diff
@@ -63,11 +64,19 @@ export class Page extends Component<Props> {
   };
 
   render() {
+    function makeLabels(title?: string): ?LabelsListProps {
+      if (!title) return null;
+
+      return { labels: [{ text: title }] };
+    }
+
     const { page, siblings, children } = this.props;
     const DateInfo = page.datePublished && (
       <HTMLDate date={new Date(page.datePublished)} />
     );
     const isLanding = page.format && page.format.id === PageFormatIds.Landing;
+    const labels = isLanding ? null : makeLabels(page.format?.title);
```

Comment:
I'd probably move this into the `const labels` check, and make this take `title: string` for ease of reading?

```suggestion
    const labels = !isLanding && page.format?.title ? makeLabels(page.format?.title) : null;
```